### DISCUSSION TITLE
Not creating a github discussion on sole reaction?

### DISCUSSION THREAD

**Discussion by Dich0tomy** [giscus/giscus#1308] [Q&A] [answered]
Hello, currently giscus creates github discussions even if someone solely reacts and doesn't leave any comments. Is there a way to disable that behaviour?

I cannot seem to find any related issue, PR or discussion, the documentation also doesn't mention such options.

Thanks from advance!

**Comment by laymonage** [comment, marked as answer]
The reactions are attached to the discussion's main post, so this is impossible unless you disable the feature completely:

<img width="741" alt="image" src="https://github.com/giscus/giscus/assets/6379424/66fe4af5-656f-4c2e-9331-8eba17e30eb9">

**Reply by Dich0tomy** [reply]
Thanks.
